sdk/src/main/java/io/opentelemetry/sdk/common/export/ConfigBuilder.java

```diff
@@ -139,4 +139,21 @@ protected static Long getLongProperty(String name, Map<String, String> map) {
       return null;
     }
   }
+
+  /**
+   * Get a double property from the map, {@code null} if it cannot be found or it has a wrong type.
+   *
+   * @param name The property name
+   * @param map The map where to look for the property
+   * @return the {@link Double} value of the property, {@code null} in case of error or if the
+   *     property cannot be found.
+   */
+  @Nullable
+  protected static Double getDoubleProperty(String name, Map<String, String> map) {
```

Comment:
We should have a unit test for this method (and for this whole class).

Reply:
Done